pysqldb3/geopackage.py

```diff
@@ -264,36 +268,28 @@ def shp_to_gpkg(self, shp_name, gpkg_tbl = None, overwrite = False, print_cmd =
             # if no gpkg_tbl name given, name the table consistente with the shape file
             gpkg_tbl = shp_name.replace('.shp', '')
 
+        # set variables
+        _overwrite = ''
+        _update = ''
+
         # check if the gpkg already exists. if so, we add another layer to the gpkg
         if not self.gpkg_exists(self.gpkg_name):
-
-            cmd = WRITE_SHP_CMD_GPKG.format(gpkg_name = self.gpkg_name,
-                                                full_path = self.path,
-                                                shp_name = shp_name,
-                                                _update = '',
-                                                gpkg_tbl = gpkg_tbl)
-        
+            _update = ''
         elif overwrite == True: # if gpkg exists and overwrite is explicityly written
-        
-            cmd = WRITE_SHP_CMD_GPKG.format(gpkg_name = self.gpkg_name,
-                                                full_path = self.path,
-                                                shp_name = shp_name,
-                                                _overwrite = '-overwrite',
-                                                gpkg_tbl = gpkg_tbl)
-
+            _overwrite = '-overwrite'
         elif not self.gpkg_tbl_exists(gpkg_tbl): # if gpkg exists and overwrite was not explicitly called
+            _update = '-update' # then add the table to the gpkg
+        elif self.gpkg_tbl_exists(gpkg_tbl): # if the gpkg and table exists but no overwrite was called
+            print("The table name to be copied to the geopackage already exists. Either change to Overwrite = True or check the name of the table to be copied.")
```

Comment:
I think this if elif conditional funnel will work, but i think explicitly checking the logic for each section will be better for long term maintenance, less likely future changes could break it. 
so you might consider the simpler approach of something like this 
```
if not overwrite and not self.gpkg_tbl_exists(gpkg_tbl:
then ...

if not overwrite and self.gpkg_tbl_exists(gpkg_tbl:
then... 
```